.eleventy.js

```diff
@@ -54,7 +54,26 @@ module.exports = (eleventyConfig) => {
     collection
       .getAll()
       .filter((item) => item.data.oss)
-      .sort((a, b) => b.date - a.date),
+      .sort((a, b) => {
+        if (a.data.feature && !b.data.feature) {
+          return -1;
+        }
+        if (b.data.feature && !a.data.feature) {
+          return 1;
+        }
+
+        if (a.data.end !== b.data.end) {
+          if (a.data.end === 'ongoing') {
+            return -1;
+          }
+          if (b.data.end === 'ongoing') {
+            return 1;
+          }
+          return b.data.end - a.data.end;
```

Comment:
Do we need to check whether `a.data.end` and `b.data.end` exist before doing this? It looks like some OSS pages (e.g. sass, true, herman, helium, django, css-remedy) don't have `end` defined at all.

Reply:
Seems like that would imply the same as "ongoing" - so we could maybe remove the explicit version, and then replace the earlier lines with eg…

`if (b.data.end === 'ongoing') { … }` => `if (!b.data.end) { … }`